CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ This project try to follows [Semantic Versioning](http://semver.org/) since the
 
 For migration information, you can always have a look at https://liip-drifter.readthedocs.io/en/latest/migrations/.
 
+## Unreleased
+
+### Added
+- Gulp role: Specify SSL certificates in BrowserSync config when role SSL is installed.
+- Gulp role: Print WebPack/Sass compilation to the browser
+
+### Changed
+- Compass role: Depreacted because Compass is no more maintained.
```

Comment:
* Typo "depreacted"
* "Compass is not maintained anymore"
* Please remove trailing `.` for consistency
* Please use present tense